CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIView**
   - Added `addGestureRecognizers(_:)` which accepts an array of `UIGestureRecognizer` to add multiple gesture recognizers to a view with one call. [#523](https://github.com/SwifterSwift/SwifterSwift/pull/523) by [moyerr](https://github.com/moyerr)
   - Added `removeGestureRecognizers(_:)` which accepts an array of `UIGestureRecognizer` to remove multiple gesture recognizers from a view with one call. [#523](https://github.com/SwifterSwift/SwifterSwift/pull/523) by [moyerr](https://github.com/moyerr)
+- **UIViewController**
+  - Added addChildViewController(_ child: , toContainerView:) which accepts a UIViewController, to easily add child view controllers.
```

Comment:
Please use back-ticks (`) for function and class names, like in the other comments.

Reply:
Sorry for that. This is my first time writing a Changelog 😅